SCHEMA-SPEC.md

```diff
@@ -24,15 +24,129 @@ None.
 
 #### Properties
 
-* `description` (optional): An informational description of the purpose of this schema.
-* `schema-url` (optional): A URL where someone may go to find more information about this schema. It is not meant for mechanical processing.
+None.
 
 #### Children
 
+* [`info`](#info-node) - one info node for that describes the schema itself.
 * [`node`](#node-node) - zero or more toplevel nodes for the KDL document this schema describes.
+* [`definitions`](#definitions-node) (optional): Definitions of nodes, values, props, and children block to reference in the toplevel nodes.
 * `node-names` (optional): [Validations](#validation-nodes) to apply to the _names_ of child nodes.
 * `other-nodes-allowed` (optional): Whether to allow nodes other than the ones explicitly listed here. Defaults to `false`.
 
+### `info` node
+
+The `info` node describes the schema itself.
+
+#### Values
+
+None.
+
+#### Properties
+
+None.
+
+#### Children
+
+* [`title`](#title-node): one or more titles
```

Comment:
Can we make all children optional here? I think requiring only specific child nodes here is a bit awkward?

Reply:
That's fair, I was coming from a more bibliographical perspective but that's not necessary here.